docs/walkthrough/run-job.md

```diff
@@ -38,6 +38,29 @@ the run status changes from:
 3. FINISHED: the job finished successfully, model has been sent to Jina AI Cloud.
 4. FAILED: the job failed, please check the logs for more details.
 
+## Continue training on new data
+
+Finetuner also supports continuing training a model produced by a previous fine-tuning run.
+If you have additional data which you want to use to further train a model, you can do this by passing its artifact id to the fit function:
+
+```python
+train_data = 'path/to/another/data.csv'
+
+run2 = finetuner.fit(
+    model='efficientnet_b0',
```

Comment:
do we still need the `model`?

Reply:
okay now i see the hint, do i understand correly that the `model` type must be identical to `artifact` model type?

Reply:
We need the model name to detect the task, so the task they are used for must be equal, so basically yes. If you use a docarray dataset, it probably does not matter